rollup.config.js

```diff
@@ -1,139 +1,69 @@
 
-// check code quality
-let eslintPlugin = eslint({
-    include: [
-        '**/*.js',
-    ],
-});
-
-// import SVG and HTML files as JS strings
-let htmlPlugin = html({
-    include: [
-        '**/*.svg',
-        '**/*.html',
-    ],
-    htmlMinifierOptions: {
-        collapseWhitespace: true,
-        quoteCharacter: "'", // reduces escape characters
-    },
-});
-
-// import LESS files as JS strings
-let lessPlugin = less({
-    options: {
-        // TODO: centralize common config settings
-        paths: [
-            'src/less',
-        ],
-    },
-});
-
-// import SASS/SCSS files as JS strings
-// see: https://sass-lang.com/documentation/js-api
-let sassPlugin = sass({
-    options: {
-        // TODO: centralize common config settings
-        includePaths: [
-            'src/scss',
-        ],
-        outputStyle: 'compact', // each style block on its own line (reduce generated white space)
-        precision: 4,
-    },
-});
-
-/* ======================================== *\
- * ROLLUP CONFIGURATION
-\* ======================================== */
+import { terserPlugin, es5plugins } from './tasks/rollup/plugins';
 
 export default [
     /* ----- BROWSER TARGETS ----- */
 
-    // src/_bundle.umd.js --> dist/*/helix-ui.browser.js (UMD)
+    // src/_bundle.es5.js --> dist/js/helix-ui.js (UMD)
     {
-        input: 'src/_bundle.umd.js',
+        input: `${srcDir}/_bundle.es5.js`,
         output: {
-            file: 'dist/scripts/helix-ui.browser.js',
+            file: pkg.unpkg,
             format: 'umd',
             name: 'HelixUI',
             sourcemap: false,
         },
         plugins: [
-            json(),
-            nodeResolve(),
-            commonjs(),
-            htmlPlugin,
-            lessPlugin,
-            sassPlugin,
-            babelPlugin,
+            ...es5plugins,
         ],
     },
 
-    // src/_bundle.umd.js --> dist/*/helix-ui.browser.min.js (UMD)
+    // src/_bundle.es5.js --> dist/js/helix-ui.min.js (UMD)
     {
-        input: 'src/_bundle.umd.js',
+        input: `${srcDir}/_bundle.es5.js`,
         output: {
-            file: 'dist/scripts/helix-ui.browser.min.js',
+            file: `${distDir}/js/helix-ui.min.js`,
             format: 'umd',
             name: 'HelixUI',
             sourcemap: false,
         },
         plugins: [
-            eslintPlugin,
-            json(),
-            nodeResolve(),
-            commonjs(),
-            htmlPlugin,
-            lessPlugin,
-            sassPlugin,
-            babelPlugin,
-            uglify({}, minify),
+            ...es5plugins,
+            terserPlugin,
         ],
     },
 
-    /* ----- NODE/BUNDLER TARGETS ----- */
+    /* ----- BUNDLER TARGETS ----- */
 
-    // src/index.js --> dist/helix-ui.js (CJS)
-    // src/index.js --> dist/helix-ui.es.js (ESM)
+    // src/_bundle.es5.js --> dist/js/helix-ui.cjs.js (CJS)
+    // src/_bundle.es5.js --> dist/js/helix-ui.esm.js (CJS)
```

Comment:
```suggestion
    // src/_bundle.es5.js --> dist/js/helix-ui.esm.js (ESM)
```

Reply:
I'll be removing this config file. It's no longer necessary.